### PR TITLE
Add binary sensors to TechnoVE integration

### PR DESCRIPTION
**Pull request by Moustachauve**
## Proposed change
This updates the documentation of the TechnoVE integration for the new binary sensors

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/108938
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by gjohansson-ST — CHANGES_REQUESTED** [review]
Atleast `ha_platforms` needs to be updated in the header section

**Comment by Moustachauve** [comment]
`ha_platforms` has been updated.

File: source/_integrations/technove.markdown [hunk 1]
```diff
@@ -19,6 +20,17 @@ The TechnoVE integration provides connectivity with TechnoVE Smart Charging Stat
 
 {% include integrations/config_flow.md %}
 
+## Binary Sensors
```
Comment (c0ffeeca7) [inline source/_integrations/technove.markdown]:
```suggestion
## Binary sensors
```
Tiny tweak. We use sentence-style capitalization also in headings.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Moustachauve and @gjohansson-ST :+1:

**Review by gjohansson-ST — APPROVED** [review]

**Pull request merged**